src/main/java/com/entropyteam/entropay/common/ReactAdminSqlMapper.java

```diff
@@ -0,0 +1,47 @@
+package com.entropyteam.entropay.common;
+
+import java.util.Map;
+import org.springframework.stereotype.Component;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Component responsible for mapping React Admin parameters to SQL parameters.
+ * It utilizes Jackson's ObjectMapper to deserialize JSON strings into Java objects.
+ */
+@Component
+public class ReactAdminSqlMapper {
+
+    private final ObjectMapper objectMapper;
+
+    public ReactAdminSqlMapper(ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+    }
+
+    /**
+     * Maps ReactAdminParams to ReactAdminSqlParams by deserializing JSON strings
+     * into Java objects and calculating SQL query parameters.
+     *
+     * @param params ReactAdminParams containing the filter, range, and sort parameters as JSON strings.
+     * @return ReactAdminSqlParams with deserialized and calculated SQL query parameters.
+     * @throws RuntimeException if JSON processing fails.
+     */
+    @SuppressWarnings("unchecked")
+    public ReactAdminSqlParams map(ReactAdminParams params) {
+        try {
+            String[] array = objectMapper.readValue(params.getSort(), String[].class);
+            String sort = array[0];
+            String order = array[1];
```

Comment:
Not a blocker, but this needs to be properly sanitized to prevent SQL Injection

Reply:
yeah, the FE is already sending the params in this specific way for all the endpoints